src/components/SecondaryPanes/Frames/tests/FrameMenu.js

```diff
@@ -0,0 +1,73 @@
+const originalL10N = L10N;
+import FrameMenu from "../FrameMenu";
+import { showMenu } from "devtools-launchpad";
+import { copyToTheClipboard } from "../../../../utils/clipboard";
+jest.mock("devtools-launchpad", () => ({ showMenu: jest.fn() }));
+jest.mock("../../../../utils/clipboard", () => ({
+  copyToTheClipboard: jest.fn()
+}));
+
+function generateMockCopySource(label, accesskey) {
+  return {
+    id: "node-menu-copy-source",
+    label: L10N.getStr(label),
+    accesskey: L10N.getStr(accesskey),
+    disabled: false,
+    // TODO: find a jest equivilent for jasmine.any
+    click: jasmine.any(Function) // eslint-disable-line no-undef
+  };
+}
+
+describe("FrameMenu", () => {
+  let mockEvent;
+  let mockFrame;
+
+  beforeEach(() => {
+    L10N = { getStr: jest.fn(value => value) };
```

Comment:
so, I left mocking the L10N in, because in our current setup, L10N returns undefined on getStr -- and this isnt meaningful enough. by mocking it, I also make sure that everything is being formatted properly.